Conflicting declarations in OpenMMCWrapper.cpp when compiling OpenMM

When compiling release `7.6.0` or the `master` branch on Arch Linux there are conflicting statements generated in `build/wrappers/OpenMMCWrapper.cpp`. I tried to compile OpenMM without cuda or opencl support.

Here are the errors that compiling gives me:
```
In file included from /home/finn/Builds/openmm/build/wrappers/OpenMMCWrapper.cpp:3:
/home/finn/Builds/openmm/build/wrappers/OpenMMCWrapper.h:177:41: error: conflicting declaration of C function ‘OpenMM_StringArray OpenMM_Platform_getPluginLoadFailures()’
  177 | extern OPENMM_EXPORT OpenMM_StringArray OpenMM_Platform_getPluginLoadFailures();
      |                                         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/finn/Builds/openmm/build/wrappers/OpenMMCWrapper.h:164:42: note: previous declaration ‘OpenMM_StringArray* OpenMM_Platform_getPluginLoadFailures()’
  164 | extern OPENMM_EXPORT OpenMM_StringArray* OpenMM_Platform_getPluginLoadFailures();
      |                                          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/finn/Builds/openmm/build/wrappers/OpenMMCWrapper.h:181:41: error: conflicting declaration of C function ‘OpenMM_StringArray OpenMM_Platform_loadPluginsFromDirectory(const char*)’
  181 | extern OPENMM_EXPORT OpenMM_StringArray OpenMM_Platform_loadPluginsFromDirectory(const char* directory);
      |                                         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/finn/Builds/openmm/build/wrappers/OpenMMCWrapper.h:163:42: note: previous declaration ‘OpenMM_StringArray* OpenMM_Platform_loadPluginsFromDirectory(const char*)’
  163 | extern OPENMM_EXPORT OpenMM_StringArray* OpenMM_Platform_loadPluginsFromDirectory(const char* directory);
      |                                          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/finn/Builds/openmm/build/wrappers/OpenMMCWrapper.cpp:246:72: error: return type ‘OpenMM_StringArray’ {aka  struct OpenMM_StringArray_struct’} is incomplete
  246 | OPENMM_EXPORT OpenMM_StringArray OpenMM_Platform_getPluginLoadFailures() {
      |                                                                        ^
/home/finn/Builds/openmm/build/wrappers/OpenMMCWrapper.cpp:246:34: error: conflicting declaration of C function  void OpenMM_Platform_getPluginLoadFailures()’
  246 | OPENMM_EXPORT OpenMM_StringArray OpenMM_Platform_getPluginLoadFailures() {
      |                                  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/finn/Builds/openmm/build/wrappers/OpenMMCWrapper.cpp:189:35: note: previous declaration ‘OpenMM_StringArray* OpenMM_Platform_getPluginLoadFailures()’
  189 | OPENMM_EXPORT OpenMM_StringArray* OpenMM_Platform_getPluginLoadFailures() {
      |                                   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/finn/Builds/openmm/build/wrappers/OpenMMCWrapper.cpp: In function ‘void OpenMM_Platform_getPluginLoadFailures()’:
/home/finn/Builds/openmm/build/wrappers/OpenMMCWrapper.cpp:248:50: error: invalid use of incomplete type ‘OpenMM_StringArray’ {aka ‘struct OpenMM_StringArray_struct’}
  248 |     return static_cast<OpenMM_StringArray>(result);
      |                                                  ^
In file included from /home/finn/Builds/openmm/build/wrappers/OpenMMCWrapper.cpp:3:
/home/finn/Builds/openmm/build/wrappers/OpenMMCWrapper.h:82:16: note: forward declaration of ‘OpenMM_StringArray’ {aka ‘struct OpenMM_StringArray_struct’}
   82 | typedef struct OpenMM_StringArray_struct OpenMM_StringArray;
      |                ^~~~~~~~~~~~~~~~~~~~~~~~~
/home/finn/Builds/openmm/build/wrappers/OpenMMCWrapper.cpp: At global scope:
/home/finn/Builds/openmm/build/wrappers/OpenMMCWrapper.cpp:261:96: error: return type ‘OpenMM_StringArray’ {aka  struct OpenMM_StringArray_struct’} is incomplete
  261 | OPENMM_EXPORT OpenMM_StringArray OpenMM_Platform_loadPluginsFromDirectory(const char* directory) {
      |                                                                                                ^
/home/finn/Builds/openmm/build/wrappers/OpenMMCWrapper.cpp:261:34: error: conflicting declaration of C function  void OpenMM_Platform_loadPluginsFromDirectory(const char*)’
  261 | OPENMM_EXPORT OpenMM_StringArray OpenMM_Platform_loadPluginsFromDirectory(const char* directory) {
      |                                  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/finn/Builds/openmm/build/wrappers/OpenMMCWrapper.cpp:185:35: note: previous declaration ‘OpenMM_StringArray* OpenMM_Platform_loadPluginsFromDirectory(const char*)’
  185 | OPENMM_EXPORT OpenMM_StringArray* OpenMM_Platform_loadPluginsFromDirectory(const char* directory) {
      |                                   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/finn/Builds/openmm/build/wrappers/OpenMMCWrapper.cpp: In function ‘void OpenMM_Platform_loadPluginsFromDirectory(const char*)’:
/home/finn/Builds/openmm/build/wrappers/OpenMMCWrapper.cpp:263:50: error: invalid use of incomplete type ‘OpenMM_StringArray’ {aka ‘struct OpenMM_StringArray_struct’}
  263 |     return static_cast<OpenMM_StringArray>(result);
      |                                                  ^
In file included from /home/finn/Builds/openmm/build/wrappers/OpenMMCWrapper.cpp:3:
/home/finn/Builds/openmm/build/wrappers/OpenMMCWrapper.h:82:16: note: forward declaration of ‘OpenMM_StringArray’ {aka ‘struct OpenMM_StringArray_struct’}
   82 | typedef struct OpenMM_StringArray_struct OpenMM_StringArray;
      |                ^~~~~~~~~~~~~~~~~~~~~~~~~
/home/finn/Builds/openmm/build/wrappers/OpenMMCWrapper.cpp: At global scope:
/home/finn/Builds/openmm/build/wrappers/OpenMMCWrapper.cpp:1345:117: error: return type ‘OpenMM_DoubleArray’ {aka ‘struct OpenMM_DoubleArray_struct’} is incomplete
 1345 | OPENMM_EXPORT OpenMM_DoubleArray OpenMM_NoseHooverChain_getYoshidaSuzukiWeights(const OpenMM_NoseHooverChain* target) {
      |                                                                                                                     ^
/home/finn/Builds/openmm/build/wrappers/OpenMMCWrapper.cpp:1345:34: error: conflicting declaration of C function ‘void OpenMM_NoseHooverChain_getYoshidaSuzukiWeights(const OpenMM_NoseHooverChain*)’
 1345 | OPENMM_EXPORT OpenMM_DoubleArray OpenMM_NoseHooverChain_getYoshidaSuzukiWeights(const OpenMM_NoseHooverChain* target) {
      |                                  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /home/finn/Builds/openmm/build/wrappers/OpenMMCWrapper.cpp:3:
/home/finn/Builds/openmm/build/wrappers/OpenMMCWrapper.h:526:41: note: previous declaration ‘OpenMM_DoubleArray OpenMM_NoseHooverChain_getYoshidaSuzukiWeights(const OpenMM_NoseHooverChain*)’
  526 | extern OPENMM_EXPORT OpenMM_DoubleArray OpenMM_NoseHooverChain_getYoshidaSuzukiWeights(const OpenMM_NoseHooverChain* target);
      |                                         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/finn/Builds/openmm/build/wrappers/OpenMMCWrapper.cpp: In function ‘void OpenMM_NoseHooverChain_getYoshidaSuzukiWeights(const OpenMM_NoseHooverChain*)’:
/home/finn/Builds/openmm/build/wrappers/OpenMMCWrapper.cpp:1347:50: error: invalid use of incomplete type ‘OpenMM_DoubleArray’ {aka ‘struct OpenMM_DoubleArray_struct’}
 1347 |     return static_cast<OpenMM_DoubleArray>(result);
      |                                                  ^
In file included from /home/finn/Builds/openmm/build/wrappers/OpenMMCWrapper.cpp:3:
/home/finn/Builds/openmm/build/wrappers/OpenMMCWrapper.h:86:16: note: forward declaration of ‘OpenMM_DoubleArray’ {aka ‘struct OpenMM_DoubleArray_struct’}
   86 | typedef struct OpenMM_DoubleArray_struct OpenMM_DoubleArray;
      |                ^~~~~~~~~~~~~~~~~~~~~~~~~
make[2]: *** [CMakeFiles/OpenMM.dir/build.make:3212: CMakeFiles/OpenMM.dir/wrappers/OpenMMCWrapper.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/Makefile2:1574: CMakeFiles/OpenMM.dir/all] Error 2
make: *** [Makefile:146: all] Error 2
```

The conflicting declarations:
```c++
/* These methods need to be handled specially, since their C++ APIs cannot be directly translated to C.
   Unlike the C++ versions, the return value is allocated on the heap, and you must delete it yourself. */
OPENMM_EXPORT OpenMM_State* OpenMM_Context_getState(const OpenMM_Context* target, int types, int enforcePeriodicBox) {
    State result = reinterpret_cast<const Context*>(target)->getState(types, enforcePeriodicBox);
    return reinterpret_cast<OpenMM_State*>(new State(result));
}
OPENMM_EXPORT OpenMM_State* OpenMM_Context_getState_2(const OpenMM_Context* target, int types, int enforcePeriodicBox, int groups) {
    State result = reinterpret_cast<const Context*>(target)->getState(types, enforcePeriodicBox, groups);
    return reinterpret_cast<OpenMM_State*>(new State(result));
}
OPENMM_EXPORT OpenMM_StringArray* OpenMM_Platform_loadPluginsFromDirectory(const char* directory) {
    vector<string> result = Platform::loadPluginsFromDirectory(string(directory));
    return reinterpret_cast<OpenMM_StringArray*>(new vector<string>(result));
}
OPENMM_EXPORT OpenMM_StringArray* OpenMM_Platform_getPluginLoadFailures() {
    vector<string> result = Platform::getPluginLoadFailures();
    return reinterpret_cast<OpenMM_StringArray*>(new vector<string>(result));
}
static char* createStringFromStream(stringstream& stream) {
    int length = stream.str().size();
    char* result = (char*) malloc(length+1);
    stream.str().copy(result, length);
    result[length] = 0;
    return result;
}
OPENMM_EXPORT char* OpenMM_XmlSerializer_serializeSystem(const OpenMM_System* system) {
    stringstream stream;
    OpenMM::XmlSerializer::serialize<OpenMM::System>(reinterpret_cast<const OpenMM::System*>(system), "System", stream);
    return createStringFromStream(stream);
}
OPENMM_EXPORT char* OpenMM_XmlSerializer_serializeState(const OpenMM_State* state) {
    stringstream stream;
    OpenMM::XmlSerializer::serialize<OpenMM::State>(reinterpret_cast<const OpenMM::State*>(state), "State", stream);
    return createStringFromStream(stream);
}
OPENMM_EXPORT char* OpenMM_XmlSerializer_serializeIntegrator(const OpenMM_Integrator* integrator) {
    stringstream stream;
    OpenMM::XmlSerializer::serialize<OpenMM::Integrator>(reinterpret_cast<const OpenMM::Integrator*>(integrator), "Integrator", stream);
    return createStringFromStream(stream);
}
OPENMM_EXPORT OpenMM_System* OpenMM_XmlSerializer_deserializeSystem(const char* xml) {
    string input(xml);
    stringstream stream(input);
    return reinterpret_cast<OpenMM_System*>(OpenMM::XmlSerializer::deserialize<OpenMM::System>(stream));
}
OPENMM_EXPORT OpenMM_State* OpenMM_XmlSerializer_deserializeState(const char* xml) {
    string input(xml);
    stringstream stream(input);
    return reinterpret_cast<OpenMM_State*>(OpenMM::XmlSerializer::deserialize<OpenMM::State>(stream));
}
OPENMM_EXPORT OpenMM_Integrator* OpenMM_XmlSerializer_deserializeIntegrator(const char* xml) {
    string input(xml);
    stringstream stream(input);
    return reinterpret_cast<OpenMM_Integrator*>(OpenMM::XmlSerializer::deserialize<OpenMM::Integrator>(stream));
}

/* OpenMM::Platform */
OPENMM_EXPORT void OpenMM_Platform_destroy(OpenMM_Platform* target) {
    delete reinterpret_cast<OpenMM::Platform*>(target);
}
OPENMM_EXPORT void OpenMM_Platform_registerPlatform(OpenMM_Platform* platform) {
    OpenMM::Platform::registerPlatform(reinterpret_cast<Platform *>(platform));
}
OPENMM_EXPORT int OpenMM_Platform_getNumPlatforms() {
    int result = OpenMM::Platform::getNumPlatforms();
    return result;
}
OPENMM_EXPORT OpenMM_Platform* OpenMM_Platform_getPlatform(int index) {
    Platform* result = &OpenMM::Platform::getPlatform(index);
    return reinterpret_cast<OpenMM_Platform*>(result);
}
OPENMM_EXPORT OpenMM_StringArray OpenMM_Platform_getPluginLoadFailures() {
    std::vector< std::string > result = OpenMM::Platform::getPluginLoadFailures();
    return static_cast<OpenMM_StringArray>(result);
}
OPENMM_EXPORT OpenMM_Platform* OpenMM_Platform_getPlatformByName(const char* name) {
    Platform* result = &OpenMM::Platform::getPlatformByName(std::string(name));
    return reinterpret_cast<OpenMM_Platform*>(result);
}
OPENMM_EXPORT OpenMM_Platform* OpenMM_Platform_findPlatform(const OpenMM_StringArray* kernelNames) {
    Platform* result = &OpenMM::Platform::findPlatform(*reinterpret_cast<const std::vector< std::string >*>(kernelNames));
    return reinterpret_cast<OpenMM_Platform*>(result);
}
OPENMM_EXPORT void OpenMM_Platform_loadPluginLibrary(const char* file) {
    OpenMM::Platform::loadPluginLibrary(std::string(file));
}
OPENMM_EXPORT OpenMM_StringArray OpenMM_Platform_loadPluginsFromDirectory(const char* directory) {
    std::vector< std::string > result = OpenMM::Platform::loadPluginsFromDirectory(std::string(directory));
    return static_cast<OpenMM_StringArray>(result);
}
OPENMM_EXPORT const char* OpenMM_Platform_getDefaultPluginsDirectory() {
    const std::string* result = &OpenMM::Platform::getDefaultPluginsDirectory();
    return result->c_str();
}
OPENMM_EXPORT const char* OpenMM_Platform_getOpenMMVersion() {
    const std::string* result = &OpenMM::Platform::getOpenMMVersion();
    return result->c_str();
}
OPENMM_EXPORT const char* OpenMM_Platform_getName(const OpenMM_Platform* target) {
    const std::string* result = &reinterpret_cast<const OpenMM::Platform*>(target)->getName();
    return result->c_str();
}
OPENMM_EXPORT double OpenMM_Platform_getSpeed(const OpenMM_Platform* target) {
    double result = reinterpret_cast<const OpenMM::Platform*>(target)->getSpeed();
    return result;
}
OPENMM_EXPORT OpenMM_Boolean OpenMM_Platform_supportsDoublePrecision(const OpenMM_Platform* target) {
    bool result = reinterpret_cast<const OpenMM::Platform*>(target)->supportsDoublePrecision();
    return (result ? OpenMM_True : OpenMM_False);
}
OPENMM_EXPORT const OpenMM_StringArray* OpenMM_Platform_getPropertyNames(const OpenMM_Platform* target) {
    const std::vector< std::string >* result = &reinterpret_cast<const OpenMM::Platform*>(target)->getPropertyNames();
    return reinterpret_cast<const OpenMM_StringArray*>(result);
}
OPENMM_EXPORT const char* OpenMM_Platform_getPropertyValue(const OpenMM_Platform* target, const OpenMM_Context* context, const char* property) {
    const std::string* result = &reinterpret_cast<const OpenMM::Platform*>(target)->getPropertyValue(*reinterpret_cast<const Context*>(context), std::string(property));
    return result->c_str();
}
OPENMM_EXPORT void OpenMM_Platform_setPropertyValue(const OpenMM_Platform* target, OpenMM_Context* context, const char* property, const char* value) {
    reinterpret_cast<const OpenMM::Platform*>(target)->setPropertyValue(*reinterpret_cast<OpenMM::Context*>(context), std::string(property), std::string(value));
}
OPENMM_EXPORT const char* OpenMM_Platform_getPropertyDefaultValue(const OpenMM_Platform* target, const char* property) {
    const std::string* result = &reinterpret_cast<const OpenMM::Platform*>(target)->getPropertyDefaultValue(std::string(property));
    return result->c_str();
}
OPENMM_EXPORT void OpenMM_Platform_setPropertyDefaultValue(OpenMM_Platform* target, const char* property, const char* value) {
    reinterpret_cast<OpenMM::Platform*>(target)->setPropertyDefaultValue(std::string(property), std::string(value));
}
OPENMM_EXPORT OpenMM_Boolean OpenMM_Platform_supportsKernels(const OpenMM_Platform* target, const OpenMM_StringArray* kernelNames) {
    bool result = reinterpret_cast<const OpenMM::Platform*>(target)->supportsKernels(*reinterpret_cast<const std::vector< std::string >*>(kernelNames));
    return (result ? OpenMM_True : OpenMM_False);
}
```

--- a/wrappers/generateWrappers.py
+++ b/wrappers/generateWrappers.py
@@ -93,7 +93,8 @@ class WrapperGenerator:
                             'bool OpenMM::Discrete1DFunction::operator!=',
                             'bool OpenMM::Discrete2DFunction::operator!=',
                             'bool OpenMM::Discrete3DFunction::operator!='
-                           ] 
+                           ]
+        self.skipMethods = [s.replace(' ', '') for s in self.skipMethods]
         self.hideClasses = ['Kernel', 'KernelImpl', 'KernelFactory', 'ContextImpl', 'SerializationNode', 'SerializationProxy']
         self.nodeByID={}
 
@@ -143,7 +144,7 @@ class WrapperGenerator:
         for section in findNodes(classNode, "sectiondef", kind="public-static-func")+findNodes(classNode, "sectiondef", kind="public-func"):
             for memberNode in findNodes(section, "memberdef", kind="function", prot="public"):
                 methodDefinition = getText("definition", memberNode)
-                if methodDefinition in self.skipMethods:
+                if methodDefinition.replace(' ', '') in self.skipMethods:
                     continue
                 methodList.append(memberNode)
         return methodList

